src/bidiMapper/bidiProtocolTypes.ts

```diff
@@ -128,11 +129,32 @@ export namespace BrowsingContext {
   export type BrowsingContextCreateType = 'tab' | 'window';
 
   export type BrowsingContextCreateParameters = {
-    type: BrowsingContextCreateType;
+    type?: BrowsingContextCreateType;
   };
+
   export type BrowsingContextCreateResult = {
     context: BrowsingContext;
   };
+
+  export namespace PROTO {
+    export type BrowsingContextNavigateCommand = {
+      method: 'PROTO.browsingContext.navigate';
```

Comment:
Since https://w3c.github.io/webdriver-bidi/#command-browsingContext-navigate exists now, does this need to be under PROTO?

Reply:
Actually I meant `PROTO.browsingContext.create`.